spacepy/pycdf/istp.py

```diff
@@ -469,12 +469,14 @@ def _validhelper(cls, v, rng=True):
         for which in (whichmin, whichmax):
             if not which in v.attrs:
                 continue
-            if v.attrs.type(which) != v.type():
+            atype = v.attrs.type(which) # attribute type
+            vtype = v.type() # variable type
```

Comment:
The comments are probably superfluous (and per PEP 8, same-line comments should have two spaces before the #). At least I'm hoping those names are clear enough that the comments are superfluous....